distributed/scheduler.py

```diff
@@ -1305,7 +1314,6 @@ def close_worker(self, stream=None, worker=None, safe=None):
     # Stimuli #
     ###########
 
-    @gen.coroutine
```

Comment:
`async` is not used here. Is this intentional?

Reply:
Probably?  Is this causing some concrete issue for you?  Does this function need to be asynchronous?